lib/setupHooks/removeReferencesToVendoredSourcesHook.sh

```diff
@@ -26,13 +26,23 @@ removeReferencesToVendoredSources() {
 
       echo -n '\)!@storeDir@/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee!g'
     )
+
+    # Sign modified binaries on aarch64-darwin
+    if [ -n "${signingUtils}" ]; then
+      signIfRequired "${installedFile}"
+    fi
```

Comment:
1. We should replace the conditional check with a new `doNotSign` variable (or similarly named) to allow it to be easily disabled if necessary
2. Similar to the other comment, we can tweak the placeholder variable to skip this stanza entirely rather than checking if `signingUtils` are set (if any derivation happens to have that variable set we might get confused here)